packages/framework/client-logger/external-telemetry/README.md

```diff
@@ -0,0 +1,115 @@
+
+### Telemetry Destinations
+
+At this time, Azure App Insights is the only available destination for Fluid external telemetry. Eventually, more destinations will be added.
+
+### What is Application Insights?
+
+At a high level, App Insights is an Azure cloud service for aggregating, visualizing, analyzing, and alerting on metrics related to a given “service” or “application”.
+You create an App Insights Instance and then configure your applications to send data to your instance using either Azure provided SDK’s or REST APIs.
+This could be general machine related health being automatically reported to the instance when you install a logging program on your service’s machines or custom metrics that you manually configure your applications to send. Keep in mind this logger is intended for use with browser based web applications, not pure nodeJS.
+In our case, we are sending custom metrics. [Learn more about Azure App Insights with their docs](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview?tabs=net)
+
+### Areas currently supported for external telemetry
+
+1. Containers
+
+# Getting Started
+
+The core functionality of this package is exposed by the `createTelemetryManagers(config: TelemetryManagerConfig);` method. A Telemetry manager handles the production and consumption/emission of telemetry events and there should be one manager created per area of interest within the Fluid Framework such as Containers. In the future more areas of interest will be added.
+
+```ts
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { FluidContainer } from "@fluidframework/fluid-static";
+import { TelemetryManagerConfig, startTelemetryManagers, createAppInsightsTelemetryConsumer } from "@fluidframework/external-telemetry"
+
+const myAppContainer: FluidContainer = {...your code to create a Fluid Continer}
+
+
+// Create App Insights Client
+const appInsightsClient = new ApplicationInsights({
+	config: {
+		connectionString:
+			"InstrumentationKey=abcdefgh-ijkl-mnop-qrst-uvwxyz6ffd9c;IngestionEndpoint=https://westus2-2.in.applicationinsights.azure.com/;LiveEndpoint=https://westus2.livediagnostics.monitor.azure.com/",
+	},
+});
+
+// Initializes the App Insights client. Without this, logs will not be sent to Azure.
+appInsightsClient.loadAppInsights();
+
+// Create the telemetry manager config object(s)
+const telemetryManagerConfig: TelemetryManagerConfig = {
+			container: myAppContainer,
+			consumers: [createAppInsightsTelemetryConsumer(appInsightsClient)],
+		};
+
+// Setup telemetry manager(s)
+startTelemetryManagers(telemetryManagerConfig);
```

Comment:
This terminology seems a bit vague to me. The intent of this function is to subscribe to Container lifetime events, right? I wonder if something like `subscribeToContainerEvents` might be more appropriate?

Reply:
Yeah I have updated this function to be called startTelemetry. We don't expose the telemetry manager concept/class so I think it shouldn't be mentioned